src/luxonis_ml/data/utils/enums.py

```diff
@@ -0,0 +1,59 @@
+from enum import Enum
+
+
+class DataLabelType(Enum):
+    """Supported computer vision label types.
+    Annotation types can be nested
+    (e.g. a BOX has 2 LABELS,
+        a BOX has a POLYLINE instance segmentation,
+        etc.)"""
+
+    CLASSIFICATION = (
+        "classification"  # used for single, multi-class, or multi-label classification
+    )
+    BOX = "box"  # bounding box
+    POLYLINE = "polyline"  # polyline to represent segmentation mask instances
+    KEYPOINTS = "keypoints"  # keypoint skeleton instances
+    LABEL = "label"  # an arbitrary label of string, bool, or number
+
+
+class MediaType(Enum):
+    """Individual file type"""
+
+    IMAGE = "image"
+    VIDEO = "video"
+    POINTCLOUD = "point cloud"
+
+
+class ImageType(Enum):
+    """Image type for IMAGE HType"""
+
+    COLOR = "color"
+    MONO = "mono"
+    DISPARITY = "disparity"
```

Comment:
Do we need to differentiate between disparity/depth if we do have mono?

Reply:
I would say yes since these images would probably be uint16. Although I'm not sure if we need depth.